feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/data/model/Fee.kt

```diff
@@ -23,10 +24,15 @@ data class EvmFee(
     override val amount = gasLimit * gasPrice
 }
 
+interface FeeInAsset : Fee {
+    val assetId: FullChainAssetId
```

Comment:
I used ids only because it's not very easy to pass Fee using fragment payload (SimpleFeeParcelModel) if we have ChainAsset model inside.

Reply:
Tbh this looks like half-way solution and will cause inconviniences for us when working with fees

Reply:
I thought that it will be helpful to have separated interface but in UI pr I removed it since it's difficult to support.

Regarding using full chain.asset model in fee: We can create ChainAssetParcel (the same way like ChainParcel using serialization) and pass it to fee payloads. What do you think?